lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -168,11 +193,17 @@ def test_interactive_backend(env, toolbar):
             pytest.skip("toolmanager is not implemented for macosx.")
     if env["MPLBACKEND"] == "wx":
         pytest.skip("wx backend is deprecated; tests failed on appveyor")
-    proc = _run_helper(_test_interactive_impl,
-                       json.dumps({"toolbar": toolbar}),
-                       timeout=_test_timeout,
-                       extra_env=env)
-
+    try:
+        proc = _run_helper(
+                _test_interactive_impl,
+                json.dumps({"toolbar": toolbar}),
+                timeout=_test_timeout,
+                extra_env=env,
+                )
+    except subprocess.CalledProcessError as err:
```

Comment:
Just to check, this changes an E result to a F result?

Reply:
I don't believe so since when an error is raised, the test results in a failure and do not go through the assert statement defined after. Therefore no `E` in any case.

Reply:
Then why add this try/except?

Reply:
This is to print out the cause of the error as printed out in `stderr` otherwise we only see a traceback of the subprocess error which is not very helpful when debugging.
P.S.: the same `try/except` is done in the `test_backend_tk.py` [tests](https://github.com/matplotlib/matplotlib/blob/main/lib/matplotlib/tests/test_backend_tk.py#L45) to extract the error statements.

Reply:
might be worth pushing that logic into our helper, but not going to hold this PR over that!